core/ui_mixins/quick_panel.py

```diff
@@ -263,6 +271,12 @@ def on_selection(self, value):
         self.value = value
         self.on_done(value)
 
+    def is_empty(self):
+        return self._is_empty
```

Comment:
I was just too lazy to implement the `@property`. Additionally, I prefer `foo.is_empty()` over `foo.is_empty`, no particular reason, just personal taste.